src/composite/MegaChart.js

```diff
@@ -88,6 +88,11 @@
         return this;
     };
 
+    MegaChart.prototype.downloadCSV = function () {
+        Utility.downloadBlob("CSV", this._chart.export("CSV"));
```

Comment:
no idea - but because this is a client side download I suspect that cannot happen?